config/locales/ur.yml

```diff
@@ -371,10 +371,10 @@ ur:
     release_date: اجراء کی تاریخ
```

Comment:
I noticed [two keys](https://github.com/alphagov/government-frontend/blob/d2789383a9117c7e54d7d18834fc945c7a1bee19/config/locales/ur.yml#L38-L39) whose values contain HTML, but the key names don't end in `_html` - were they missed?

Reply:
This confused me for a second! 😆 Those keys are actually unused and have actually been removed in the the following commit: https://github.com/alphagov/government-frontend/pull/2305/commits/45d2065a38d14f8bad40409fe7903613b51deb3b#diff-f8ecf4cd17cdd1ecc6f81401e44eba0a0e3c55ff9cd30b5acd1a3c4029fc5638L38-L39

Reply:
Great - thanks for confirming!